src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static Map<Fruit, Integer> storage = new HashMap<>();
+
+    public Map<Fruit, Integer> getStorage() {
+        return storage;
+    }
```

Comment:
if the `storage` field itself is public, maybe we can remove this method?

Reply:
Fixed